packages/graphql/codegen.ts

```diff
@@ -4,7 +4,10 @@ import type { Types } from '@graphql-codegen/plugin-helpers'
 const commonGenerateOptions: Types.ConfiguredOutput = {
   config: {
     reactQueryVersion: 5,
-    fetcher: '../client#fetcher',
+    fetcher: {
+      func: '../client#fetcher',
+      isReactHook: false,
```

Comment:
is this only for fetcher?  with this option do we still generate hooks?

Reply:
It's just for the fetcher. I ran codegen a few times since and we still have hooks but going to double check

Reply:
It's just for the fetcher. I ran codegen a few times since and we still have hooks but going to double check